packages/2019-disaster-game/src/components/atoms/NewBadge.js

```diff
@@ -59,70 +54,68 @@ const titleText = css`
   font-size: 14rem;
 `;
 
-const NewBadge = ({ type, badges, playAudio, stopAudio }) => {
-  const [badgeInfo, setBadgeInfo] = useState(badges[type]);
+const NewBadge = ({
+  badgeData,
+  displayBadgeTime = 11,
+  playAudio,
+  stopAudio,
+  finishDisplayCallback,
+  fade = true
+}) => {
+  const [badgeInfo, setBadgeInfo] = useState(badgeData);
   const [hideBadge, setHideBadge] = useState(false);
+  const fadeBadgeTimeout = (displayBadgeTime - 1) * 1000;
```

Comment:
Should this timing be pulled out to live with other timing variables?

Reply:
Which variables are you thinking of? This modal gets used across a couple of chapters

Reply:
I guess it's getting calculated based on displayBadgeTime, which is being passed in as a prop, so it's fine